### ISSUE TITLE
Figure out the Licensing to use for our project

### ISSUE THREAD

**Issue by TroyKomodo** [ScuffleTV/scuffle#9]
There are many opensource licensing versions we can pick from.
I basically want anyone to be able to run a streaming platform, use code from here and anything. The only thing I do not want is for someone to use code from here and run a commercial product off of this repo. 
I do not want a big company to find us and then go ahead and spin us up on their stack and sell a product. 
I do not want a competitor to steal our works and publish it.

**Comment by 4w3Official** [comment]
perhaps [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode)?

**Comment by IS2511** [comment]
The license in commit 60f6a05d047df5545f42b4f199f45e097b8f1e96 literally contains copyright for some random company in 1991. Also it says version 2 in it and the commit says 3.
Can we please choose a license from https://choosealicense.com/appendix/?
I assume you want to prohibit big companies from commercial use, but you'll need to make an exception for yourself. Related: https://opensource.stackexchange.com/questions/4875/open-source-license-to-prevent-commercial-use
Basically, it should be possible to achieve what you want, but with much effort. The general strategy seems to be [open-source license like GNU GPLv3] for non-commercial use, contributor agreement to maintain ownership of the whole project, separate commercial license.
It's also of note that prohibiting commercial use might be against point 6 of the OSD https://opensource.org/osd-annotated

**Comment by IS2511** [comment]
Also, if you'll be making money on this, what about the other contributors? 🤨

**Comment by treuks** [comment]
> The license in commit [60f6a05](https://github.com/ScuffleTV/scuffle/commit/60f6a05d047df5545f42b4f199f45e097b8f1e96) literally contains copyright for some random company in 1991. 

That's the FSF, not some random company.
